u2f-core/src/response.rs

```diff
@@ -108,6 +109,10 @@ impl Response {
                 // Status word [2 bytes]
                 StatusCode::UnknownError.write(&mut bytes);
             }
+            Response::Bogus => {
+                // Status word [2 bytes]
+                StatusCode::NoError.write(&mut bytes);
```

Comment:
This causes Chrome and Firefox to happily continue on, other values could cause infinite loops of trying the bogus registration or loop trying authentication again even though the relevant keyhandle is not registered.